scripts/3_visualize/visualizeRelativeAbundance.R

```diff
@@ -38,25 +39,35 @@ visualizeRelativeAbundance <- function(tag='desktop', file.in, file.text, target
     fill <- group$col
     cx <- as.numeric(group$cx)
     cy <- as.numeric(group$cy)
-    
+    cr <- 70
     offset.x <- -50
     offset.y <- -30
     line.spacing <- 16
     off.x <- -55
     off.y <- -10
     opacity <- '0.2'
     
+    on.right <- cx > 300 # on right side of bars
+    
     g <- dinosvg:::svg_node("g", svg, c(id=group$name))
+    
+    if (on.right){
+      # top of circle to top of rect, to bottom of rect, to bottom of circle
+      d = sprintf("M%s %s a%s,%s 0 0,0 -%s,%s L 300 %s L 300 %sz", cx, cy-cr, cr,cr,cr,cr, start.y+height, start.y)
```

Comment:
So....I see what you are doing. I'm guessing this is all we'll have time for, but if I got insomnia...is there any sort of spline-type path or curve or something that would allow more flexibility?